apps/prairielearn/python/prairielearn/sympy_utils.py

```
@@ -196,11 +251,11 @@ class _SympyJsonStrPrinter(StrPrinter):
     Callers must deserialize with `allow_sets=True` or the literal forms will be rejected.
     """
 
-    def _print_EmptySet(self, expr: sympy.Set) -> str:  # pyright: ignore[reportIncompatibleMethodOverride] # ruff:ignore[unused-method-argument, invalid-function-name]
-        return "{}"
+    def _print_EmptySet(self, expr: sympy.Set) -> str:  # ruff:ignore[unused-method-argument, invalid-function-name]
```

Ruff (0.16.2)

[warning] Invalid rule code in suppression: invalid-function-name, unused-method-argument

Enable `lint.preview` to use rule names

(RUF102)